lib/es/management/commands/reindex.py

```diff
@@ -140,13 +143,16 @@ def handle(self, *args, **kwargs):
             modules.append('stats')
 
         if kwargs.get('wipe', False):
-            confirm = raw_input('Are you sure you want to wipe all AMO '
-                                'Elasticsearch indexes? (yes/no): ')
+            noconfirm = kwargs.get('noconfirm', False)
+            confirm = ''
+            if not noconfirm:
```

Comment:
Small nit maybe change this var just so the condition reads better. E.g. skip_confirmation or similar.

```python
if not skip_confirmation:
    # whatever
```
reads better than:

```python
if not noconfirm:
      # whatever
```

Alternatively you could rename the option (Sorry, my bad for suggesting something that doesn't work will in conditionals!).